### ISSUE TITLE
training loss is high

### ISSUE THREAD

**Issue by TongWeiDP** [touristCheng/UCSNet#5]
when I use your pre-trained model for training,the initialization loss is high for 200+,what's the question about this? and loss is continuous high during the several epochs,the batch_size is 1,other parameters is not changed,Looking forward to your reply,Thanks!

**Comment by Airobin329** [comment]
the same question, why?

**Comment by atztao** [comment]
the same issue ....
